include/ovs_control.h

```diff
@@ -12,144 +12,157 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
+#pragma once
 #ifndef OVS_CONTROL_H
 #define OVS_CONTROL_H
 
 #define IPTOS_PREC_INTERNETCONTROL 0xc0
 #define DSCP_DEFAULT (IPTOS_PREC_INTERNETCONTROL >> 2)
-#define STDOUT_FILENO   1   /* Standard output.  */
+#define STDOUT_FILENO 1 /* Standard output.  */
 
 #include <openvswitch/vconn-provider.h> /* add to /usr/local/include/openvswitch */
+#include <openvswitch/namemap.h>
 #include <openvswitch/ofpbuf.h>
 #include <openvswitch/ofp-errors.h>
-#include <openvswitch/ofp-packet.h>
-#include <openvswitch/ofp-port.h>
-#include <openvswitch/ofp-table.h>
-#include <openvswitch/ofp-protocol.h>
-#include <openvswitch/ofp-switch.h>
-#include <openvswitch/ofp-flow.h>
+#include <openvswitch/ofp-util.h>
+#include <openvswitch/ofp-parse.h>
+//#include <openvswitch/ofp-packet.h>
+//#include <openvswitch/ofp-port.h>
+//#include <openvswitch/ofp-table.h>
+//#include <openvswitch/ofp-protocol.h>
+//#include <openvswitch/ofp-switch.h>
+//#include <openvswitch/ofp-flow.h>
 
-extern "C" { 
-    struct unixctl_conn;
+extern "C" {
+struct unixctl_conn;
```

Comment:
@lfu-ps If we don't use `vconn`, we probably don't need this struct too.

Reply:
I'm testing removing the entire ovs_control.h/cpp, but currently keeping it for function reference (even it's not invoked).